lib/models/balance.model.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/material.dart';
+import 'package:ten_ten_one/models/amount.model.dart';
 
 class BalanceModel extends ChangeNotifier {
-  // TODO: Change to 0, just to see that it updates :)
-  int amount = -1;
```

Comment:
I am surprised that the tests do not fail after that change.

Reply:
That is indeed surprising! 
I played locally with it and got it failing - but only after I changed the test to use a different unexpected value (I changed it to `1000`); after that it also failed with `-1`. Afterwards changing it to `0` made it pass as expected.

I think somehow the state is not properly re-built? Do we cache things for the test?
@klochowicz any idea what could cause this?

Reply:
I'll try to play with it to see what's happening with it.